packages/build_cmd/lib/repo_finder.dart

```diff
@@ -0,0 +1,103 @@
+import 'dart:io' as io;
+import 'package:args/args.dart';
+import 'package:args/command_runner.dart';
+import 'package:path/path.dart' as path;
+import 'package:io/ansi.dart';
+
+/// Mixin to help [Command] subclasses locate both `google/mediapipe` and
+/// the root of `google/flutter-mediapipe` (this repository).
+///
+/// The primary methods are [findFlutterMediaPipeRoot] and [findMediaPipeRoot].
+///
+/// By default, the root for `google/flutter-mediapipe` is determined by the
+/// firest ancestor directory which contains a `.flutter-mediapipe-root` file
+/// (whose contents are irrelevant), and the root of `google/mediapipe` is
+/// expected to be a sibling of that. However, the `--source` flag can overwrite
+/// this expectation and specify an absolute path where to find `google/mediapipe`.
```

Comment:
For what reason?

Reply:
Developers may not have these repositories as siblings on their machines, and forcing that via this command feels presumptuous. WDYT?

Reply:
A note on the `Outdated` flag - this code has not changed, only moved directories. The above conversation still applies as-is.